Dockerfile

```diff
@@ -157,6 +158,7 @@ RUN apt-get update && \
     chown authentik:authentik /certs /media /authentik/.ssh /ak-root
 
 COPY ./authentik/ /authentik
+COPY ./packages/ /packages
```

Comment:
This isn't enough. Docker fails with `ModuleNotFoundError: No module named 'django_dramatiq_postgres'`